src/hooks/useStore.js

```diff
@@ -80,6 +94,28 @@ const useStore = create(set => ({
 			podcasts: state.podcasts.filter(podcast => podcast.key !== state.selectedPodcast.key),
 		}));
 	},
+	tags: defaultTags,
+	addTagToPodcast: (tag, podcastToTag) => {
+		console.log(tag, podcastToTag.title);
+		set(state => ({
+			podcasts: state.podcasts.map(podcast => {
+				if (podcast.key === podcastToTag.key) {
+					return {...podcast, tags: [...podcast.tags, tag]};
+				}
+				return podcast;
+			}),
+		}));
+	},
+	removeTagFromPodcast: (tag, podcastToUntag) => {
+		set(state => ({
+			podcasts: state.podcasts.map(podcast => {
+				if (podcast.key === podcastToUntag.key) {
+					return {...podcast, tags: podcast.tags.filter(t => t !== tag)};
```

Comment:
I suggest renaming `tag` to `selectedTag` or something similiar, so you can use `tag` instead of `t`. It is best not to use abbreviations.

Reply:
I knew you will comment on this. ;-) That's what I get for being lazy.